modules/microshift-install-rpms-olm.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-install-rpm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-installing-with-olm-from-rpm-package_{context}"]
+== Installing the Operator Lifecycle Manager (OLM) from an RPM package
+
+When you install {microshift-short}, the Operator Lifecycle Manager (OLM) package is not installed by default. You can install the OLM on your {microshift-short} instance using a RPM package. 
+
+.Prerequisites
+
+* You have created a {microshift-short} instance. 
+
+.Procedure 
+
+. Install the OLM package by running the following command: 
++
+[source,terminal]
+----
+$ dnf install microshift-olm
```

Comment:
Should this include `sudo` before the command? Seems to be the pattern in other parts of documentation (same for `systemctl` in other places)

Reply:
Yes, great catch! Its often done more in MicroShift then in OCP, so Ill fix that!